test/e2e/cluster_config.sh

```diff
@@ -41,6 +41,8 @@ clusters:
     routegroups_validation: "enabled"
     stackset_routegroup_support_enabled: "true"
     stackset_ingress_source_switch_ttl: "1m"
+    spotio_account_id: "${SPOTIO_ACCOUNT_ID}"
```

Comment:
How would it work in E2E, where we have multiple clusters? Can they coexist together just fine?

Reply:
Yeah, that's not a problem, in the former implementation we had one ocean per node pool, now we just have one per cluster, as they are named after the cluster and each have their own spotinst controller running it should work fine.